services/collaboration/pkg/server/http/server.go

```diff
@@ -182,5 +182,16 @@ func prepareRoutes(r *chi.Mux, options Options) {
 				})
 			})
 		})
+		r.Route("/templates/{templateID}", func(r chi.Router) {
+			r.Use(
+				func(h stdhttp.Handler) stdhttp.Handler {
+					// authentication and wopi context
+					return colabmiddleware.WopiContextAuthMiddleware(options.Config, h)
+				},
+			)
```

Comment:
I think the `CollaborationTracingMiddleware` should also be used here.

Reply:
Correct! Thanks.